applications/main/nfc/nfc_app_i.h

```diff
@@ -93,6 +97,12 @@ typedef struct {
     bool is_key_attack;
     uint8_t key_attack_current_sector;
     bool is_card_present;
+    uint8_t nested_phase;
+    uint8_t prng_type;
+    uint8_t backdoor;
```

Comment:
These three variables hold values from enum types, so it would be nice to change their types to appropriate enums

Reply:
@RebornedBrain Fixed: https://github.com/flipperdevices/flipperzero-firmware/pull/3822/commits/a7c0819034fdbe591d64d16b95ba1f88cff660eb